service/src/main/kotlin/app/cash/backfila/ui/actions/BackfillCreateHandlerAction.kt

```diff
@@ -55,11 +60,23 @@ class BackfillCreateHandlerAction @Inject constructor(
       createRequestBuilder.parameter_map(customParameters)
     }
 
-    val response = createBackfillAction.create(
-      service = formFields[BackfillCreateField.SERVICE.fieldId]!!,
-      variant = formFields[BackfillCreateField.VARIANT.fieldId]!!,
-      request = createRequestBuilder.build(),
-    )
+    val response = try {
+      createBackfillAction.create(
+        service = formFields[BackfillCreateField.SERVICE.fieldId]!!,
+        variant = formFields[BackfillCreateField.VARIANT.fieldId]!!,
+        request = createRequestBuilder.build(),
+      )
+    } catch (e: BadRequestException) {
```

Comment:
Wouldn't this be a cash most/all errors situation?

Reply:
Yes, and then it would show them in the UI. Since this action only handles form submissions from the UI, I think that is the desired functionality.